src/main/java/essenmakanan/recipe/RecipeIngredientList.java

```diff
@@ -1,4 +1,64 @@
 package essenmakanan.recipe;
 
+import essenmakanan.ingredient.Ingredient;
+import essenmakanan.parser.IngredientParser;
+
+import essenmakanan.exception.EssenFormatException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class RecipeIngredientList {
+    private ArrayList<Ingredient> ingredients = new ArrayList<>();
+
+    public RecipeIngredientList() {
+        Scanner in = new Scanner(System.in);
+        String input;
+        boolean isAddingIngredients = true;
+
+        do {
+            System.out.println("Add ingredient (name,quantity,unit) of your recipe, type \"end\" to finish");
+            input = in.nextLine();
+
+            if (input.equals("end")) {
+                isAddingIngredients = false;
+            } else {
+                assert (input != null) : "Input is null";
+                this.addIngredient(input);
+            }
+        } while (isAddingIngredients);
```

Comment:
Nice that there is an alternative that will prompt user to input steps or ingredients when missing